applications/blink_raw/targets/freertos.mips4k.pic32mx.lccbuffer/nbproject/Makefile-default.mk

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
It seems there are a bunch of generated files in the application target / nbproject directory that are specific to your editor and there are paths here that are specific to your computer.
Try adding "nbproject" to the .gitignore file and removing these files from the repository.